addons/sale_expense/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,18 @@
+---Models in module 'sale_expense'---
+---Fields in module 'sale_expense'---
+
+---XML records in module 'sale_expense'---
+DEL ir.model.access: sale_expense.access_sale_order_employee
+# NOTHING TO DO: removed from manifest
+
+DEL ir.rule: sale_expense.sale_order_rule_expense_user (noupdate)
+# DONE: post-migration: try to delete
+
+NEW ir.ui.view: sale_expense.assets_backend
+NEW ir.ui.view: sale_expense.hr_expense_form_view_inherit_account_manager
+NEW ir.ui.view: sale_expense.hr_expense_form_view_inherit_saleman
+NEW ir.ui.view: sale_expense.qunit_suite
+# NOTHING TO DO: new xmls
+
+DEL product.product: hr_expense.product_product_fixed_cost (noupdate)
+# NOTHING TO DO: moved from one data file to another
```

Comment:
Move from where? Seeing this, it seems it has to be removed as well

Reply:
yeah, it has to be removed as well

Reply:
This data has been moved to demo file. So yes it can be removed. But should I remove from hr_expense instead? I will also migrate hr_expense to fix this then.

Reply:
@MiquelRForgeFlow why it hasn't been recognized as moved from other module?

Reply:
because demo data is not used in analysis

Reply:
@bishalgit the hr_expense is already migrated in https://github.com/OCA/OpenUpgrade/pull/2439, which you can review

Reply:
OK, I see, I would let then `hr_expense` and try here to remove it: if not possible, it has been used already.

Reply:
@MiquelRForgeFlow and @pedrobaeza I will try to delete here as well then. Thanks